addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -55,104 +55,16 @@ def test_channel_members(self):
         self.assertEqual(channel.message_partner_ids, self.env['res.partner'])
         self.assertEqual(channel.channel_partner_ids, self.env['res.partner'])
 
-        emp_partner_write_date = fields.Datetime.to_string(self.env.user.partner_id.write_date)
         test_partner_write_date = fields.Datetime.to_string(self.env.user.partner_id.write_date)
 
         def get_add_member_bus():
-            message = self.env["mail.message"].search([], order="id desc", limit=1)
             member = self.env["discuss.channel.member"].search([], order="id desc", limit=1)
             return (
                 [
                     (self.cr.dbname, "res.partner", self.test_partner.id),
                     (self.cr.dbname, "discuss.channel", channel.id),
-                    (self.cr.dbname, "discuss.channel", channel.id, "members"),
-                    (self.cr.dbname, "discuss.channel", channel.id),
-                    (self.cr.dbname, "discuss.channel", channel.id),
                 ],
                 [
-                    {
-                        "type": "mail.record/insert",
-                        "payload": {
-                            "discuss.channel": [
-                                {"id": channel.id, "last_interest_dt": "2020-03-22 10:42:06"},
-                            ],
-                        },
-                    },
-                    {
-                        "type": "mail.record/insert",
-                        "payload": {"discuss.channel": [{"id": channel.id, "is_pinned": True}]},
-                    },
-                    {
-                        "type": "discuss.channel/new_message",
-                        "payload": {
-                            "data": {
-                                "mail.message": self._filter_messages_fields(
-                                    {
-                                        "attachment_ids": [],
-                                        "author_guest_id": False,
-                                        "author_id": {
-                                            "id": self.env.user.partner_id.id,
-                                            "type": "partner",
-                                        },
-                                        "body": [
-                                            "markup",
-                                            f'<div class="o_mail_notification" data-oe-type="channel-joined">invited <a href="#" data-oe-model="res.partner" data-oe-id="{self.test_partner.id}">@Test Partner</a> to the channel</div>',
```

Comment:
Likewise, this would remove the only notification test we have in mail about this one.
We might want to test on a group too here.

And maybe introduce another test which specifically check there is no join/leave message in channel.